core/db_models/EEM_Form_Element.model.php

```diff
@@ -163,7 +164,7 @@ protected function __construct(FormStatus $form_status, InputTypes $input_types,
                     'FIN_type',
                     esc_html__('Form element type.', 'event_espresso'),
                     false,
-                    false,
+                    null,
```

Comment:
This caused that `array_key_exists` error and thus the [test failures](https://github.com/eventespresso/event-espresso-core/runs/2955033784?check_suite_focus=true)